src/FLARM/Hardware.hpp

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright The XCSoar Project
+
+#pragma once
+
+#include "NMEA/Validity.hpp"
+#include "util/StaticString.hxx"
+
+#include <type_traits>
+
+/**
+ * The FLARM hardware read-out from PFLAC config sentences.
+ */
+struct FlarmHardware {
+  Validity available;
+
+  NarrowString<64> device_type;
+  NarrowString<64> capabilities;
+
+  bool isPowerFlarm() noexcept {
```

Comment:
PowerFlarm supports additional baud rates, and has 2 data ports:

Reply:
Yes, true. It seems to work with just one baud input as well (sets both ports at once). If you want, I can include the other one as well. There are even more PowerFlarm (and ADSB) related config items as well as possible read-outs of the FLARM ID and others, but I wanted to get the basics through first.